routes/productRoute.js

```diff
@@ -15,7 +19,11 @@ const productRoute = (router) => {
   router.get('/', getAllProducts);
 
   router.get('/:id', validate(getProductSchema), getProduct);
+
+  router.delete('/delete/:id', validate(deleteProductSchema), deleteP);
```

Comment:
. lets change URL to : /:id instead of delete/:id
. Change functionName to deleteProduct